Makefile

```diff
@@ -51,7 +51,7 @@ gotest:
 
 jsonnettest: ks $(JSONNET_FILES)
 #	TODO: use `ks check` once implemented
-	./ks -J lib show -f $(KCFG_TEST_FILE) -f $(GUESTBOOK_FILE) >/dev/null
+	cd testdata/test-app && ../../ks -J lib show default -c $(KCFG_TEST_FILE) -c $(GUESTBOOK_FILE) >/dev/null
```

Comment:
What do you think about calling one or the other `test-data` or `testapp` for consistency?